content/reference/core-configuration.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Core configuration'
+categories: ["reference"]
+description: Configuration for the Rhize core
+weight: 900
+menu:
+  main:
+    parent: reference
+    identifier:
+---
+
+ Core acts as a bespoke edge agent, overseeing data sources such as OPC-UA servers, and managing the publication and subscription of topics within the NATS messaging system.
```

Comment:
```suggestion
The Core service oversees data sources such as OPC-UA servers and manages the publication and subscription of topics within the NATS messaging system.
```

Reply:
Simplifiying to remove any confusion with the Agent service, and most of the Core behavior is really interesting only internally, so no need to explain too much.